tests/src/test/scala/cats/tests/NonEmptyVectorTests.scala

```diff
@@ -218,4 +222,40 @@ class NonEmptyVectorTests extends CatsSuite {
       }
     }
   }
+
+  test("NonEmptyVector#hashCode consistent with Vector#hashCode") {
+    forAll { (nonEmptyVector: NonEmptyVector[Int]) =>
+      nonEmptyVector.hashCode should === (nonEmptyVector.toVector.hashCode)
+    }
+  }
+
+  test("NonEmptyVector#equals consistent with Vector#equals") {
+    forAll { (lhs: NonEmptyVector[Int], rhs: NonEmptyVector[Int]) =>
+      lhs.equals(rhs) should === (lhs.toVector.equals(rhs.toVector))
+    }
+  }
+
+  test("NonEmptyVector#toString produces correct output") {
+    forAll { (nonEmptyVector: NonEmptyVector[Int]) =>
+      nonEmptyVector.toString should === (s"NonEmpty${nonEmptyVector.toVector.toString}")
+    }
+    NonEmptyVector(1, Vector.empty).toString should === ("NonEmptyVector(1)")
+    NonEmptyVector(1, Vector.empty).toVector.toString should === ("Vector(1)")
+  }
+
+  test("Cannot create a new NonEmptyVector from constructor") {
+    if(Platform.isJvm) {
+      if (!Properties.versionNumberString.startsWith("2.10")) {
+        // A bug in scala 2.10 allows private constructors to be accessed.
+        // We should still ensure that on scala 2.11 and up we cannot construct the
+        // object directly. see: https://issues.scala-lang.org/browse/SI-6601
+        "val bad: NonEmptyVector[Int] = new NonEmptyVector(Vector(1))" shouldNot compile
+      }
+    }
+  }
+
+  test("Cannot create a new NonEmptyVector from apply with an empty vector") {
```

Comment:
Yea, that's true - :-). I didn't get a chance to get back to this unfortunately. I'll get to it, and the other comments, by the end of the week all going well.